smakarov/07_css/chess.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html> 
+<HTML> 
+        <HEAD> 
+            <META charset="utf-8"> 
+            <TITLE>Пример страницы</TITLE>
+            <style type="text/css">
+            	th, table {border: 1px solid grey; border-collapse: collapse;}
+            	table {width:800px;}
+            	th {height:100px; font-size: 50px}
+            	tr:nth-child(odd) th:nth-child(even) {background: black; color: white;}
```

Comment:
Here can be like:
tr:nth-child(odd) th:nth-child(even),
tr:nth-child(even) th:nth-child(odd) {}